copilot-bootstrap.py

```diff
@@ -48,7 +48,7 @@
             Optional("backing-services"): [
                 {
                     "name": str,
-                    "type": lambda s: s in ("s3", "external-s3", "postgres", "redis", "opensearch",),
+                    "type": lambda s: s in ("s3", "external-s3", "aurora-pg", "rds-pg", "redis", "opensearch",),
```

Comment:
I think this is not up to date with the `main` branch, is it? I believe we now use `s3-policy` rather than `external-s3`.